<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapperPathInfo</class>
 <widget class="QWidget" name="MapperPathInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MapperPathInfo 
{
	background-color: &quot;#404040&quot;;
	color: &quot;#DBDBDB&quot;;

	border-radius: 24;
}

/*
* 		  PATH TITLE
*/
MapperPathInfo #path_name_lbl
{
	color:  &quot;#A480ED&quot;;
}


/*
* 		  plain text edit
*/
MapperPathInfo #full_path_pte
{
	background-color:  &quot;#404040&quot;;
	border: 0 solid transparent;
}


/*
* 		  BUTTONS
*/
MapperPathInfo #change_path_btn,
MapperPathInfo #check_path_btn,
MapperPathInfo #open_path_btn
{
	background-color: transparent;
	color:  &quot;#DBDBDB&quot;;

	border-radius: 4;
	border: 1px solid  &quot;#DBDBDB&quot;;

	padding-left: 4;
	padding-right: 4;
	padding-top: 8;
	padding-bottom: 8;
}


MapperPathInfo #change_path_btn:hover,
MapperPathInfo #check_path_btn:hover,
MapperPathInfo #open_path_btn:hover
{
	background-color: rgba(255, 255, 255, .15);
}

MapperPathInfo #change_path_btn::pressed,
MapperPathInfo #check_path_btn::pressed,
MapperPathInfo #open_path_btn::pressed
{
	background-color: rgba(0, 0, 0, .15);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="path_type_icon">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>file type icon</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="full_path_vlay">
         <property name="spacing">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="path_name_lbl">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>52</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>18</pointsize>
             <fontweight>DemiBold</fontweight>
            </font>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string>ThisIsAVeryVeryVeryVeryVeryVeryVeryVery VeryVeryVeryVeryVery VeryVery</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="full_path_pte">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>52</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>C:\Users\test\OneDrive\Documenti\Desktop\Sparking zero tournament ITA.png</string>
           </property>
           <property name="maximumBlockCount">
            <number>2</number>
           </property>
           <property name="centerOnScroll">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="state_hlay">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="state_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>16</pointsize>
           <fontweight>DemiBold</fontweight>
          </font>
         </property>
         <property name="text">
          <string>Path State:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttons_vlay">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="open_path_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Open Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_path_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Change Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="check_path_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Check State</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="v_sp">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="openable_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>16</pointsize>
         <fontweight>DemiBold</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Openable:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
